sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRepositoryClientLiveTests.cs

```diff
@@ -109,6 +110,84 @@ public async Task CanSetRepositoryProperties()
             // Cleanup
             await client.SetPropertiesAsync(originalContentProperties);
         }
+        #endregion
+
+        #region Tag Tests
+        [RecordedTest]
+        public async Task CanGetTags()
+        {
+            // Arrange
+            var client = CreateClient();
+
+            // Act
+            AsyncPageable<TagProperties> tags = client.GetTagsAsync();
+
+            bool gotV1Tag = false;
+            await foreach (TagProperties tag in tags)
+            {
+                if (tag.Name.Contains("v1"))
+                {
+                    gotV1Tag = true;
+                }
+            }
+
+            // Assert
+            Assert.IsTrue(gotV1Tag);
+        }
+
+        [RecordedTest]
+        public async Task CanGetTagsWithCustomPageSize()
+        {
+            // Arrange
+            var client = CreateClient();
+            int pageSize = 2;
+            int minExpectedPages = 2;
+
+            // Act
+            AsyncPageable<TagProperties> tags = client.GetTagsAsync();
+            var pages = tags.AsPages(pageSizeHint: pageSize);
+
+            int pageCount = 0;
+            await foreach (var page in pages)
+            {
+                Assert.IsTrue(page.Values.Count <= pageSize);
+                pageCount++;
+            }
+
+            // Assert
+            Assert.IsTrue(pageCount >= minExpectedPages);
```

Comment:
suggestion: The `Assert.That(..)` syntax is for comparisons is a bit nicer when they fail as you get a more descriptive error.

For example:
```c#
Assert.That(pageCount, Is.GreaterThanOrEqualTo(minExpectedPages));
```

Reply:
There is also an Assert.GreaterOrEqual